src/Core/Maintenance/Staging/Command/SystemSetupStagingCommand.php

```diff
@@ -34,7 +34,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new ShopwareStyle($input, $output);
 
-        if (!$io->confirm('This command will install the Shopware 6 system in staging mode. It will overwrite existing data in this database, make sure you use a staging database and have a backup', false)) {
```

Comment:
i would like to keep the default here to false as the consequences can be quite drastic

however what do you think about either explicitly checking the --no-interaction flag here instead? or alternatively add a special --confirm flag, so that the confirm question will be skipped if the flag is set?

Reply:
@keulinho 
I have solved it via the --force option, I took it from the system:install command